### PR TITLE
Add iOS and tvOS frameworks as well as unit test support for both platforms

### PR DESCRIPTION
**Pull request by jbeenie**

### CODE REVIEW

File: PathKit/PathKit_tvOS/PathKit_tvOS.h [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+//
+// This file is part of PathKit. The full PathKit copyright notice,
+// including terms governing use, modification, and redistribution, is
+// contained in the file LICENSE at the root of the source code distribution
+// tree.
+
+#import <UIKit/UIKit.h>
+
+//! Project version number for PathKit_tvOS.
+FOUNDATION_EXPORT double PathKit_tvOSVersionNumber;
```
Comment (alejandro-isaza) [inline PathKit/PathKit_tvOS/PathKit_tvOS.h]: We can resume the same header for every platform.

File: PathKit/PathKit_tvOS/Info.plist [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
```
Comment (alejandro-isaza) [inline PathKit/PathKit_tvOS/Info.plist]: We can reuse the same info.plist for every platform.

File: PathKit/PathKit_iOSTests/PathKit_iOSTests.swift [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testExample() {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+    }
+
+    func testPerformanceExample() {
```
Comment (alejandro-isaza) [inline PathKit/PathKit_iOSTests/PathKit_iOSTests.swift]: We can reuse the same tests for every platform.

**Review by alejandro-isaza — CHANGES_REQUESTED** [review]

File: PathKit/PathKit_tvOS/PathKit_tvOS.h (hunk 1 above)
Comment (jbeenie) [reply PathKit/PathKit_tvOS/PathKit_tvOS.h]: Did you mean reuse the same header? If so I'm not sure how to do that. Can you show me how? Or send me a resource that shows how to do that?

Comment (alejandro-isaza) [reply PathKit/PathKit_tvOS/PathKit_tvOS.h]: There is nothing to it. Just delete duplicates and add to all targets as a public header.

Comment (jbeenie) [reply PathKit/PathKit_tvOS/PathKit_tvOS.h]: Ah my bad, I was over complicating things. Thought there was more to it.

**Review by alejandro-isaza — APPROVED** [review]

**Pull request merged**